src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -40,6 +40,28 @@ static char* _int2str(int num)
 }
 
 
+static void _assign(LottieSlot::Pair *pair, LottieProperty::Type type)
+{
+    delete(pair->prop);
+
+    switch (type) {
+        case LottieProperty::Type::ColorStop: {
+            pair->prop = static_cast<LottieGradient*>(pair->obj)->colorStops.copy();
```

Comment:
it looks copy() doesn't necessary:
```
pair->prop = &static_cast<LottieGradient*>(pair->obj)->colorStops;
```

Reply:
Without copy(), I think the shallow copy operator will change the value, and it will effect slot reverting in this code.
Or would changing shallow copy operator to deep copy approach be better?